Resolve this error in the build by including sys/sysmacros.h in
qga/commands-posix.c:

  /build/source/qga/commands-posix.c:668: undefined reference to `major'
--- a/qga/commands-posix.c
--- b/qga/commands-posix.c
@@ -14,6 +14,7 @@
 #include <glib.h>
 #include <sys/types.h>
 #include <sys/ioctl.h>
+#include <sys/sysmacros.h>
 #include <sys/wait.h>
 #include <unistd.h>
 #include <errno.h>
